### PR TITLE
Add `setLength` method to FunctionTemplate

### PR DESCRIPTION
**Pull request by clover2123**
Signed-off-by: HyukWoo Park <hyukwoo.park@samsung.com>

### CODE REVIEW

**Review by ksh8281 — APPROVED** [review]

File: src/runtime/FunctionTemplate.cpp [hunk 1]
```diff
@@ -127,10 +127,18 @@ void FunctionTemplate::updateCallbackFunction(FunctionTemplateRef::NativeFunctio
 
 void FunctionTemplate::setName(AtomicString name)
 {
+    // should be called before instantiate
```
Comment (bbrto21) [inline src/runtime/FunctionTemplate.cpp:130]: please move this comment to header file

File: src/runtime/FunctionTemplate.cpp [hunk 2]
```diff
@@ -127,10 +127,18 @@ void FunctionTemplate::updateCallbackFunction(FunctionTemplateRef::NativeFunctio
...
     ASSERT(m_cachedObjectStructure == nullptr);
     m_name = name;
 }
 
+void FunctionTemplate::setLength(size_t length)
+{
+    // should be called before instantiate
```
Comment (bbrto21) [inline src/runtime/FunctionTemplate.cpp:137]: ditto

File: src/runtime/FunctionTemplate.cpp (hunk 1 above)
Comment (clover2123) [reply src/runtime/FunctionTemplate.cpp]: Thanks for the review. Fixed it.

**Review by bbrto21 — APPROVED** [review]
LGTM

**Pull request merged**